client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -1,544 +1,644 @@
+                key={VOTE_TOTALS_TAB_ID}
+                panel={
+                  <BatchResultTallySheet
+                    areResultsFinalized={areResultsFinalized}
+                    cancelSheetRename={cancelSheetRename}
+                    confirmSheetRename={confirmSheetRename}
+                    contest={contest}
+                    deleteSheet={() => deleteSheet(0)}
+                    enableEditing={enableEditing}
+                    formMethods={formMethods}
+                    index={0}
+                    initiateSheetRename={() => initiateSheetRename(0)}
+                    isEditing={isEditing}
+                    isRenamingSheet={isRenamingSheet}
+                    isTotalsSheet={sheets.length > 1}
+                    savedResults={
+                      selectedBatch.resultTallySheets[0]?.results || {}
+                    }
+                    saveResults={saveResults}
+                    sheets={sheets}
+                  />
+                }
+              >
+                Vote Totals
+              </Tab>
+              {sheets.length > 1 &&
+                sheets.map((sheet, i) => (
+                  <Tab
+                    id={sheet.name}
+                    key={sheet.id}
+                    panel={
+                      <BatchResultTallySheet
+                        areResultsFinalized={areResultsFinalized}
+                        cancelSheetRename={cancelSheetRename}
+                        confirmSheetRename={confirmSheetRename}
+                        contest={contest}
+                        deleteSheet={() => deleteSheet(i)}
+                        enableEditing={enableEditing}
+                        formMethods={formMethods}
+                        index={i}
+                        initiateSheetRename={() => initiateSheetRename(i)}
+                        isEditing={isEditing}
+                        isRenamingSheet={isRenamingSheet}
+                        savedResults={
+                          selectedBatch.resultTallySheets[i]?.results || {}
+                        }
+                        saveResults={saveResults}
+                        sheets={sheets}
+                      />
+                    }
+                  >
+                    {sheetNameBeforeRename && sheetNameBeforeRename.index === i
+                      ? sheetNameBeforeRename.name
+                      : watch(`resultTallySheets[${i}].name`)}
+                  </Tab>
+                ))}
+
+              <Tabs.Expander />
+
+              {isEditing && (
+                <Button
+                  disabled={isRenamingSheet}
+                  icon="add"
+                  minimal
+                  onClick={addSheet}
+                >
+                  {sheets.length > 1
+                    ? 'Add sheet'
+                    : 'Use multiple tally sheets'}
+                </Button>
+              )}
+            </Tabs>
+          </>
+        )}
+      </Detail>
```

Comment:
I forgot one important feature when handing this off to you: We need to show who last updated the batch tally (JM or tally entry user)! Let's discuss where to add this in

Reply:
Ah yes I noticed this in the WIP branch you handed over! Happy to discuss